### DISCUSSION TITLE
OpenSSL New Governance Structure: A Call to the Academic Community

### DISCUSSION THREAD

**Discussion by romen** [openssl/openssl#24988] [Announcements]
# OpenSSL New Governance Structure: A Call to the Academic Community

Hello OpenSSL Communities,

I am personally thrilled about the [recent announcement][blog_post] from OpenSSL about its new governance structure. I am particularly excited as I think these changes are a significant step forward towards involving more and diverse voices and experiences in the project, in pursuit of the [OpenSSL Mission][mission]:

> We believe everyone should have access to security and privacy tools, whoever they are, wherever they are or whatever their personal beliefs are, as a fundamental human right.

## New Governance Framework and Board of Directors

I think this diagram is worth a thousand words in summarizing the details presented in the blog post and press release:

![OpenSSL Boards of Directors][bod]

The Communities interested in OpenSSL are now truly at the root of the new governance structure for both the OpenSSL Foundation and the OpenSSL Corporation. I personally believe this will bring mutual benefits to individuals and organizations within these communities, as well as to OpenSSL itself.

## Community Advisory Committees

![Community Advisory Committees][advisory_committees]

To gather and amplify the voices of the OpenSSL Communities, the Project is establishing, for each of the two new entities, a couple of Advisory Committees: a Business Advisory Committee (BAC) and a Technical Advisory Committee (TAC).

With members elected from within the Communities, these Committees will provide a direct channel for collecting input and diverse perspectives from the rich ecosystem of the OpenSSL communities!

One of the most exciting aspects of these changes, for me, is that they offer new paths for individuals and organizations throughout the OpenSSL Communities to engage directly with project governance.
Whereas before, the main path to get involved always started with external developers distinguishing themselves and being invited to become committers, and later, possibly, joining the OpenSSL Technical Committee or the OpenSSL Management Committee, I look forward to non-developers and more people with a broader set of interests, experiences, and insights engaging with OpenSSL!

## New Projects under the OpenSSL Mission

It's also worth highlighting that the OpenSSL Library is not the only project to adopt the [OpenSSL Mission and Values][mission].
The announcement today revealed that [Bouncy Castle][BC] and [cryptlib][cryptlib] have also adopted them, and I look forward to more projects electing to do the same in the future.

As these projects collaborate together towards the same mission, all our communities will benefit through innovation, improvements, and tackling common challenges more effectively.

Even though support from the OpenSSL Foundation and Corporation is on a opt-in voluntary basis for projects adopting the OpenSSL Mission, I expect that the more active engagement of OpenSSL communities through discussions and the Advisory Committees will ultimately mutually benefit all the projects adopting the Mission, even those not directly availing themselves of the support of the new OpenSSL Governance Structure.

## Looking Forward to the Future

The future looks incredibly promising with these new changes.
As a member of the academic community deeply involved in OpenSSL, I see immense potential for these updates to drive new mutually beneficial research and development opportunities in the fields of cryptography, security, and privacy.

I also firmly believe that these changes, and the engagement of all our diverse communities, will ultimately drive a positive, long-lasting, and pervasive impact in all of our lives.

## Invitation to the Academic Community

To my fellow researchers:
This is an exciting time to be involved in OpenSSL.
These changes offer the promise of a fresh avenue for collaboration, experimentation, and research. While sharing my excitement for these announcements, I also personally want to encourage all in the academic community who are interested in the [OpenSSL Mission and Values][mission] to engage with these updates, share your insights, and contribute to new and ongoing discussions.

I expect that participating in the ongoing and upcoming discussions, and ultimately contributing to the advisory committees for both the OpenSSL Foundation and Corporation, will be beneficial for all of us.

I believe our academic community can affect change and provide insight both in the Technical and the Business Advisory Committees, enriching the discourse with perspectives that until now were underrepresented in the governance of the project.

### Join the Conversation

Let’s come together to explore these new changes, share our perspectives, and collaboratively shape the future of OpenSSL.
Whether you have technical questions, theoretical insights, practical suggestions, or any other kind of feedback, your participation is invaluable.

Feel free to leave a comment here, start new topics, or join existing discussions. Together, we can push the boundaries of what’s possible with OpenSSL as well as other security and privacy projects, and contribute to their continued evolution.

The first concrete step in this process is for our academic community to self-organize and find ways to facilitate discussions within members of our community.

In pure academic style, I offer more open questions than concrete definitive solutions, and I look forward to engaging with all of you!

Best regards,

Nicola Tuveri  
Doctoral Researcher  
Tampere University  
FINLAND

[blog_post]: https://openssl-library.org/post/2024-07-24-openssl-new-governance-structure/
[bod]: https://openssl-library.org/images/blog/openssl_journey_BOD.png
[mission]: https://openssl-mission.org/
[advisory_committees]: https://openssl-library.org/images/blog/openssl_journey_BAC.png
[BC]: https://www.bouncycastle.org/
[cryptlib]: https://cryptlib.com/

**Comment by romen** [comment]
@bbbrumley @cryptojedi @dstebila @baentsch @SWilson4 @praveksharma can you help me spread this further?

**Reply by romen** [reply]
This blog post provides instructions on how to join a forum for OpenSSL Communities: https://openssl-library.org/post/2024-10-30-join-bacs/

**Comment by rsbeckerca** [comment]
Question on Nominations: the link appears to only work on Chrome (not FireFox), which is fine. I cannot find the Election Committee is to invite.

**Reply by romen** [reply]
We could probably ask @mattcaswell about this.

**Reply by arapov** [reply]
@rsbeckerca, you should now be able to invite the Election Committee. Which link are you referring to that isn’t working in Firefox?

**Comment by romen** [comment]
This blog post provides instructions on how to join a forum for OpenSSL Communities: https://openssl-library.org/post/2024-10-30-join-bacs/
